src/main/views/webContentEvents.js

```diff
@@ -136,6 +136,12 @@ const generateNewWindowListener = (getServersFunction, spellcheck) => {
             return;
         }
 
+        // Image proxy case
+        if (parsedURL.pathname.match(/^\/api\/v[3-4]\/image/)) {
+            shell.openExternal(url);
+            return;
+        }
+
```

Comment:
maybe we should send to `openExternal` anything that comes from the API? is there anything in API that should be displayed on the desktop?

Reply:
I think we're blocking anything else that comes from the API, or it opens in a popup window. Not sure what the popup case is. I'd prefer to just handle individual cases as they come up and block other stuff unless it makes sense to allow it through. Just to reduce other potential security problems.